### DISCUSSION TITLE
Docker Stops Upon Full Sync

### DISCUSSION THREAD

**Discussion by thedinz** [Taxel/PlexTraktSync#734] [Q&A] [answered]
For some reason the docker container stops every time a library sync is complete, restarting it is fruitless as it will perform another sync and then stop again.

How can I keep this running?

Nothing in the logs about this except the completion of the sync

dated plex watchlist in 0.0 seconds
mpleted full sync in 4 min 14.3 seconds
exTraktSync [0.18.9]

**Comment by imjuzcy** [comment, marked as answer]
By default, it runs a sync and stops. Are you looking for the `watch` command? The `sync` command is meant to run a sync and stop when it's done, and you are supposed to run it periodically (with cron etc) and not continuously.

**Reply by thedinz** [reply]
Ok that make sense i figure it would run constantly and look for any changed, the watch command i [mistakenly] thought only applied to the scrobbler aspect.

**Reply by thedinz** [reply]
Could you tell me how do set that part up? I on unraid, so when i start the container i open the console it simple shows /app # and when i type in either plextraktsync watch it says not found, sorry i am not that good with this

**Reply by glensc** [reply]
There's unraid section in readme, did you read it?
- https://github.com/Taxel/PlexTraktSync#unraid-setup

**Reply by thedinz** [reply]
I sure did, that's how i got where i am now, however that small blurb doesn't help me getting the watch command working, although, it may help some others, for me its lacking information on how to get it going,

**Reply by glensc** [reply]
If you combine it with watch documentation you should be able to deduct `python3 -m plextraktsync watch`:
- https://github.com/Taxel/PlexTraktSync#watch

**Reply by thedinz** [reply]
Yeap, make sense, I was thinking that but was unsure if it would roll me through the credential process again as I already did that. The more you know....

**Reply by glensc** [reply]
In that case, You should have noticed the credential process asked you questions, so you can ctrl+c to abort if you don't want it again...

**Reply by glensc** [reply]
Anyway, added the `plextraktsync` to docker image as well:
- https://github.com/Taxel/PlexTraktSync/pull/736

**Reply by thedinz** [reply]
I understand it must be infuriating helping newbies. But one day, I will be able to help others. This newbie is still stuck, after running the command it seems incomplete, as in if I close the console it says I will lose my changed and it does indeed fail to stick, I have attached a screenshot
![ScreenShot_20220120143953](https://user-images.githubusercontent.com/68015411/150410696-181c1941-7b33-4708-ba2a-518b9402f16f.png)
.

**Reply by glensc** [reply]
the screenshot proves you managed to start watch command successfully.  keep it open and start playing something on plex,  you should see some messages and trakt.tv site showing something is playing.

this is the point I can't guide you further, I have no idea wtf is unraid other than it seems to use docker images nor do I have or will ever have access to the windows system.

if you want ask community question about how to get watch running in the background you should post a new question with proper title so it's visible/searchable properly. your current title doesn't even mention unraid, even it's clearly unraid specific. and don't forget to cross link things!

the problem with helping newbies is that they lose interest at the point they got their problem solved, they never contribute back. haven't seen any pr's (even if asked multiple times) to improve readme. so that's very demotivating.

**Reply by imjuzcy** [reply]
> I understand it must be infuriating helping newbies. But one day, I will be able to help others. This newbie is still stuck, after running the command it seems incomplete, as in if I close the console it says I will lose my changed and it does indeed fail to stick, I have attached a screenshot ![ScreenShot_20220120143953](https://user-images.githubusercontent.com/68015411/150410696-181c1941-7b33-4708-ba2a-518b9402f16f.png) .

First and foremost, I do not use unRAID, but I think it has support for `docker-compose`. You have to figure out how to use it since I do not have access to unRAID, after that you can just use the `docker-compose` example in the README, with one additional line `command: watch` in the `docker-compose.yml` file. With that, when you run `docker-compose up -d` the container will start detached (in the background) with the `watch` command. And when you close the terminal window it'll keep running in the background.

You can verify it with `docker logs [ne of the container]`, it'll show you exactly what you have in the screenshot.

**Reply by thedinz** [reply]
> the problem with helping newbies is that they lose interest at the point they got their problem solved, they never contribute back. haven't seen any pr's (even if asked multiple times) to improve readme. so that's very demotivating.

Sadly there is a lot of truth to this, but its not all of us....
